lesson36/src/js/article.js

```diff
@@ -0,0 +1,114 @@
+const articleList = document.getElementById("js-article-List");
+
+const createElementWithClassName = (type, className) => {
+  const element = document.createElement(type);
+  element.className = className;
+  return element;
+};
+
+const createErrorMessage = (text) => {
+  const errorMessage = createElementWithClassName("p", "article-error-message");
+  errorMessage.textContent = text;
+  return errorMessage;
+};
+
+const renderLoading = () => {
+  const loadingWrapper = createElementWithClassName("div", "article-loading-wrapper");
+  const loading = document.createElement("img");
+  loadingWrapper.id = "js-loading";
+  loading.src = "./img/article-loading.gif";
+  articleList.appendChild(loadingWrapper).appendChild(loading);
+};
+
+const removeLoading = () => document.getElementById("js-loading").remove();
+
+const fetchData = async (endpoint) => {
+  const response = await fetch(endpoint);
+  if (!response.ok) {
+    const responseMessage = `${response.status}:${response.statusText}`;
+    articleList.append(createErrorMessage(responseMessage));
+    console.error(responseMessage);
+    return;
+  }
+  return await response.json();
+}
```

Comment:
I was just working on an implementation here,
and I thought it would be nice to have the versatility!
- creating a p with createErrorMessage()
- and then appending it to each one!
Thank you!!